File: 61.c
#include<stdio.h>
 
int main(){
 
    int a, b, x1 = 0, x2 = 0;
     
    for(int i = 0; i < 8; i+=2){
        scanf("%d %d", &a, &b);
        x1+=a; x2+=b;
    }
 
    if(x1 > x2){
        printf("1");
    }
    else if(x1 < x2){
        printf("2");
    }
    else{
        printf("DRAW");
    }
 
    return 0;
}

File: 61.py
one = input().split()
two = input().split()
three = input().split()
four = input().split()
 
for i in range(2):
    one[i] = int(one[i])
for i in range(2):
    two[i] = int(two[i])
for i in range(2):
    three[i] = int(three[i])
for i in range(2):
    four[i] = int(four[i])
 
game_one = one[0] + two[0] + three[0] + four[0]
game_two = one[1] + two[1] + three[1] + four[1]
 
if game_one > game_two:
    print(1)
elif game_one < game_two:
    print(2)
else:
    print("DRAW")
